From 102dd300004b26428cab14684c71e8659b54c1ed Mon Sep 17 00:00:00 2001
From: Nikolaus Rath <Nikolaus@rath.org>
Date: Tue, 23 Feb 2016 16:30:36 -0800
Subject: Use 'cython3' instead of 'cython' command

Origin: debian
Forwarded: not-needed

In Debian, python3-cython provides only a 'cython3' command. To avoid
pulling in the complete Python 2.x runtime environment for
python-cython, we use 'cython3' instead.
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index b7f1c..9513b 100755
--- a/setup.py
+++ b/setup.py
@@ -205,7 +205,7 @@ class build_cython(setuptools.Command):
 
     def run(self):
         try:
-            version = subprocess.check_output(['cython', '--version'],
+            version = subprocess.check_output(['cython3', '--version'],
                                               universal_newlines=True,
                                               stderr=subprocess.STDOUT)
         except OSError:
@@ -215,7 +215,7 @@ class build_cython(setuptools.Command):
         if not hit or LooseVersion(hit.group(1)) < "0.24":
             raise SystemExit('Need Cython 0.24 or newer, found ' + version)
 
-        cmd = ['cython', '-Wextra', '--force', '-3', '--fast-fail',
+        cmd = ['cython3', '-Wextra', '--force', '-3', '--fast-fail',
                '--directive', 'embedsignature=True', '--include-dir',
                os.path.join(basedir, 'Include'), '--verbose' ]
         if DEVELOPER_MODE:
